doc/api/net.md

```diff
@@ -2,9 +2,14 @@
 
 > Stability: 2 - Stable
 
-The `net` module provides you with an asynchronous network wrapper. It contains
-functions for creating both servers and clients (called streams). You can include
-this module with `require('net');`.
+The `net` module provides an asynchronous network API for creating stream-based
+servers ([`net.Server`][]) and clients ([`net.Socket`][]) that implement TCP
```

Comment:
I'll just put `net.createServer` and `net.createConnection` then (the classes are not usually `new`ed anyway :D)

On a side note, what is the preferred way to create a client? `net.createConnection` and `net.connect` are just aliases now..(`net.createConnection` does look better when you put it alongside `net.createServer` though)